Dockerfile

```diff
@@ -1,9 +1,9 @@
 # gitbox with gitlist v.1.0.1
-# https://github.com/nmarus/docker-gitbox
-# Nicholas Marus <nmarus@gmail.com>
+# https://github.com/pvdlg/docker-gitbox
+# Pierre Vanduynslager <pierre.denis.vanduynslager@gmail.com>
 
 FROM debian:jessie
-MAINTAINER Nicholas Marus <nmarus@gmail.com>
+MAINTAINER Pierre Vanduynslager <pierre.denis.vanduynslager@gmail.com>
```

Comment:
```suggestion
```
`MAINTAINER` has been deprecated, so lets remove to align with https://github.com/semantic-release/docker-gitbox/pull/3#discussion_r817942324